iphone/Maps/UI/PlacePage/Components/PlacePageHeader/PlacePageHeaderViewController.swift

```diff
@@ -50,7 +50,35 @@ extension PlacePageHeaderViewController: PlacePageHeaderViewProtocol {
     }
   }
 
-  func setTitle(_ title: String) {
-    titleLabel?.text = title
+  func setTitle(_ title: String?, secondaryTitle: String?) {
+    guard let title = title,
+          let titleFont = StyleManager.shared.theme?.fonts.medium20 else {
```

Comment:
Is it possible that there is valid title, bit titleFont is not there? Then the title won't be displayed.

Reply:
Let's crash immediately instead of guarding if the font is not available. Then we can easily discover and fix it while testing.

Reply:
Ok, changed to force-unwrap style.